src/pages/Chat/components/ChatRoom/ChatRoom.tsx

```diff
@@ -62,6 +76,73 @@ function ChatRoom({ userSelfId, userRole }: IChatRoom) {
 
   const ref = useChatScroll(messages);
 
+  const isClient = userRole === UserRoleEnum.CLIENT;
+
+  const renderSendOfferButton = (): JSX.Element => {
```

Comment:
IMHO, if you use switch here this part of code will shorter. And DRY

Reply:
fixed